.github/workflows/docker-build.yml

```diff
@@ -84,6 +84,8 @@ jobs:
     steps:
     - name: Checkout Repository
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      with:
+        fetch-depth: 0
```

Comment:
Would `fetch-tags: true` work as well?

Reply:
I guess [not](https://github.com/actions/checkout/issues/1471).

Reply:
I saw it as well but then found https://github.com/actions/checkout/issues/1471, so I just stick with the same config as in the release job.